smart_campus/app/templates/app/question_add_page.html

```diff
@@ -0,0 +1,89 @@
+{% extends 'app/base.html' %}
+{% load static %}
+
+{% block stylesheet %}
+  {{ block.super }}
+  <link rel="stylesheet" type="text/css" href="{% static "app/css/edit.css" %}">
+{% endblock %}
+
+{% block content %}
+<div class="ui text container">
+  <form class="ui big borderless form" action="" method="POST" id="question_form">
+    {% csrf_token %}
+    <div class="required five wide field">
+      <label>題目內容</label>
+      <input type="text" name="content">
+    </div>
+    <div class="grouped fields">
+      <label>選項及答案勾選</label>
+      <div class="inline required fields">
+        <div class="field">
+          <div class="ui toggle checkbox">
+            <input type="radio" name="answer" value="1" checked="cheked">
+            <label>選項一</label>
+          </div>
+        </div>
+        <div class="required five wide field">
+          <input type="text" name="choice_contents">
+        </div>
+      </div>
+      <div class="inline required fields">
+          <div class="field">
+            <div class="ui toggle checkbox">
+              <input type="radio" name="answer" value="2">
+              <label>選項二</label>
+            </div>
+          </div>
+          <div class="required five wide field">
+            <input type="text" name="choice_contents">
+          </div>
+        </div>
+      <div class="inline required fields">
+        <div class="field">
+          <div class="ui toggle checkbox">
+            <input type="radio" name="answer" value="3">
+            <label>選項三</label>
+          </div>
+        </div>
+        <div class="required five wide field">
+          <input type="text" name="choice_contents" >
+        </div>
+      </div>
+      <div class="inline required fields">
+        <div class="field">
+          <div class="ui toggle checkbox">
+            <input type="radio" name="answer" value="4">
+            <label>選項四</label>
+          </div>
+        </div>
+        <div class="required five wide field">
+          <input type="text" name="choice_contents">
+        </div>
+      </div>
```

Comment:
line 19 ~ line 36 in question_edit_page.html should be written in a template and be reused here (line 20 ~ line 61).

Reply:
Sorry, I forgot to....
I will fix it.